dispatcher/config_test.ini

```diff
@@ -0,0 +1,60 @@
+[DEFAULT]
+
+# Poll frequency in seconds for main program loop. It makes
+# some sense to make other time-based options multiples of
+# this, though not required
+PollFrequency = 3
+
+# How long to wait for boards to arm
+ArmTimeout = 40
+
+# How long since a client's last check-in until we consider
+# it to be 'timing out'
+ClientTimeout = 15
+
+# Database URIs. Please store DB password in the MONGO_PASSWORD
+# environment variable and runs DB password as RUNS_MONGO_PASSWORD
+ControlDatabaseURI = mongodb://daq:%%s@gw:27020/admin
+ControlDatabaseName = daq
+RunsDatabaseURI = mongodb://daq:%%s@gw:27017/admin
+RunsDatabaseName = xenonnt
+RunsDatabaseCollection = runs
+
+# Timeouts (seconds to wait until we assume it failed)
+# Give Arm command some time in case baselines noisy
+ArmCommandTimeout = 60
+# Give start command less time because there's not really anything complicated to do
+StartCommandTimeout = 20
+# Give stop longer in general in case we're writing off a big buffer
+StopCommandTimeout = 30
+# Also retry stop/reset n times before throwing alarm this one is not in seconds
+# but is the number of times to retry (seconds_before_alarm = n*StopCommandTimeout)
+RetryReset = 3
+# Min time between Stop-Arm and Arm-Start commands
+TimeBetweenCommands = 6
+
+# these are the control keys to look for
+ControlKeys = active comment link_mv link_nv mode softstop stop_after
+
+# Declare detector configuration here. Each detector's top level
+# key is its system-wide name identifier. Under the top-level
+# hosts are fields for the readers and crate controller.
+MasterDAQConfig = {
+        "tpc": {
+            "controller": ["reader0_controller_0"],
+            "readers": [
+                "reader0_reader_0",
+                "reader1_reader_0",
+                "reader2_reader_0",
+                "reader3_reader_0"
+            ]
+        },
+        "muon_veto": {
+            "controller": ["reader5_controller_0"],
+            "readers": ["reader5_reader_0"]
+        },
+        "neutron_veto": {
+            "controller": ["reader6_controller_0"],
+            "readers":    ["reader6_reader_0", "reader6_reader_1"]
+        }
+    }
```

Comment:
This is the production config

Reply:
Do you mean the readers and the controller hosts?
And what will be the test config change in this case?

Reply:
```
"tpc": {"controller": ["reader4_controller_0"], "readers": ["reader4_reader_0"]},
"muon_veto": {"controller": ["reader4_controller_1"], "readers": ["reader4_reader_1"]},
"neutron_veto": {"controller": ["reader4_controller_2"], "readers": ["reader4_reader_2"]}
```